crates/ruff_linter/src/settings/types.rs

```diff
@@ -513,6 +513,7 @@ pub enum SerializationFormat {
     Github,
     Gitlab,
     Pylint,
+    Rdjson,
```

Comment:
I changed this name so that the command-line accepts `rdjson` rather than `rd-json`. (There are other ways to achieve this, but Rust says initializations should be treated as one word anyway.)